tests/deployment/test_base.py

```diff
@@ -255,3 +255,7 @@ async def test_find_all_flows_in_dir_tree(self, project_dir):
                 ),
             },
         ]
+
+    async def test_find_all_flows_works_on_large_directory_structures(self):
+        flows = await _search_for_flow_functions(str(prefect.__development_base_path__))
+        assert len(flows) > 500
```

Comment:
and perhaps tests for those exceptions here?

Reply:
Added a test ensuring that `_find_flow_functions_in_file` doesn't raise when encountering an error while opening a file.